ldaptor/samba/smbpassword.py

```diff
@@ -1,9 +1,15 @@
-import string, warnings
+import warnings
+try:
+    import string
+    maketrans = string.maketrans
+except ImportError:
```

Comment:
While this does not gives errors, this is broken as here we should catch AttributeError

I am fixing it a branch which is a child of a child of this one so to simplify the merge  I think that is ok to merge it as it is... as with the squash merge things will get messy